core/communication/port-authority/package.json

```diff
@@ -21,6 +21,7 @@
     "@polypoly-eu/bubblewrap": "file:../bubblewrap",
     "@types/body-parser": "^1.19.0",
     "@types/connect": "^3.4.33",
+    "@types/mocha": "^9.0.0",
```

Comment:
Are these actually needed, if Mocha is not being used anymore? And do they have to be a _dependency_, as opposed to _devDependency_ now?

Reply:
In this specific case, mocha is really used. I'll check about that.